src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/mapView.js

```diff
@@ -23,34 +23,40 @@ styleHead.innerText = `.mapboxgl-map {
         }`
 document.head.appendChild(styleHead);
 
+const TOKEN = 'pk.eyJ1IjoiZW5zby1vcmciLCJhIjoiY2tmNnh5MXh2MGlyOTJ5cWdubnFxbXo4ZSJ9.3KdAcCiiXJcSM18nwk09-Q';
+
+const GEO_POINT         = "GeoPoint";
+const GEO_MAP           = "GeoMap";
+const SCATTERPLOT_LAYER = "ScatterplotLayer";
```

Comment:
The names are improper - I mean they work now only because the libsare defined wrong. It sohuld be `Geo_Point`, `Geo_Map`, and `Scatterplot_Layer`. Talk with @sylwiabr to know more and agree how to fix it.